app/src/main/java/org/greenstand/android/TreeTracker/usecases/CreateTreeUseCase.kt

```diff
@@ -34,10 +34,11 @@ class CreateTreeUseCase(
             wallet = ""
         )
         analytics.treePlanted()
-        val attributeEntitites = params.treeCaptureAttributes().map {
-            TreeAttributeEntity(it.key, it.value, -1)
-        }.toList()
-        Timber.d("Inserting TreeCapture entity $entity")
+        val attributeEntitites: List<TreeAttributeEntity>? = params.treeCaptureAttributes().entries.map {
+            TreeAttributeEntity("${it.key}", "${it.value}" , -1)
+        }
+
```

Comment:
What was this part changed for?

Reply:
It was actually not necessary, I have replaced it with the existing one.